app/facades/quotes_facade.rb

```diff
@@ -0,0 +1,8 @@
+class QuotesFacade
+  def self.searched_quotes(keyword)
```

Comment:
This looks good for getting the quote attributes that we want but what about the `:totalCount`?? Here is a way that you can get both with a single call:
```
def self.searched_quotes(keyword)
    json = QuotesService.searched_quotes(keyword)
    @quotes = json[:results].map do |quote_data|
      Quote.new(quote_data)
    end
    [json[:totalCount], @quotes]
end
```
Now we can give our controller access to both of these as well